tweakextras

```diff
@@ -68,7 +77,7 @@ def main():
 
 
 def ids_from_stdin():
-    for ln in sys.stdin.read():
+    for ln in sys.stdin.readlines():
```

Comment:
Ah.  Then I would agree.  The problem with read() was that it was a block
read, and did not return a list.   So I was getting one character at a time.

I'm assuming that sys.stdin returns one line at a time, right?

On Thu, Jul 23, 2015 at 11:32 AM, Ian Ward <notifications@github.com> wrote:

> In tweakextras
> <https://github.com/open-data/tweakextras/pull/1#discussion_r35334613>:
>
> > @@ -68,7 +77,7 @@ def main():
> >
> >
> >  def ids_from_stdin():
> > -    for ln in sys.stdin.read():
> > +    for ln in sys.stdin.readlines():
>
> hmm. I'll take it but even better is
>
> for ln in sys.stdin:
>
> because that streams the input
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/open-data/tweakextras/pull/1/files#r35334613>.
>